docs/docs/secret/scanning.md

```diff
@@ -92,9 +92,29 @@ If you don't need secret scanning, you can disable it via the `--security-checks
 $ trivy image --security-checks vuln alpine:3.15
 ```
 
+## Recommendation
+We would recommend specifying `--skip-dirs` for faster secret scanning.
+In container image scanning, Trivy walks the file tree rooted  `/` and scans all the files other than [built-in allowed paths][builtin].
```

Comment:
Would be easier to add a line number to the link, altho it will change frequently... 😢

Reply:
Yes, I'm worried that the reference will be stale soon. Then, users will refer to the old file. Any idea?

Reply:
I split builtin.go into builtin-rules.go and builtin-allow-rules.go.